packages/houdini-utils/src/paint/fallback/util/wrapper.ts

```diff
@@ -1,7 +1,11 @@
 const createWrapper = (id: string): HTMLElement => {
+  // TODO: fix global
```

Comment:
can we create issues and ref it here to have clear actionable for team owning this package ?

Reply:
Added #183, will push a change updating the comments to reference it.